docs-site/docs/user/project/projects/bot-accounts/bot-accounts-intro.md

```diff
@@ -4,4 +4,4 @@ id: introduction
 
 # Bot Accounts
 
-<!-- TODO: Bot accounts intro -->
+Bot accounts are to be used when performing actions which can't be attributed to a human user. For example, using an uploader.
```

Comment:
Remove the `to be` from this sentence and add in `launching automated workflows` to the example